### PR TITLE
Override synced fields

### PR DESCRIPTION
**Pull request by kaedroho**

### CODE REVIEW

**Review by KalobTaulien — APPROVED** [review]
Overall this looks good. I made some minor comments, as usual. 

One thing I _think_ I didn't notice was ListBlock support. That's non-blocking for this PR but I'm curious if that's coming, or if I just missed it somewhere.

**Comment by kaedroho** [comment]
> One thing I think I didn't notice was ListBlock support. That's non-blocking for this PR but I'm curious if that's coming, or if I just missed it somewhere.

This issue came up again for the commenting work Jacob is doing. We're working on an RFC to add stable IDs for these blocks (the approach has already got Matthew's approval in principle, we just need to finish writing the RFC!)

File: wagtail_localize/migrations/0010_overridablesegment.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        ('wagtail_localize', '0009_stringtranslation_errors'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='OverridableSegment',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('order', models.PositiveIntegerField()),
+                ('data_json', models.TextField()),
```
[the remark this answers is not in the document]
Comment (kaedroho) [reply wagtail_localize/migrations/0010_overridablesegment.py:19]: We're keeping Django 2.11 support for now as that is what Mozilla are currently using (it's hard for them to upgrade).

File: wagtail_localize/models.py [hunk 2]
```diff
@@ -1038,6 +1074,55 @@ def from_value(cls, template_value):
...
+
+    def get_error(self):
+        """
+        Returns a string containing any validation errors on the saved value.
+        """
+        # Check if a database error was raised when we last attempted to publish
+        if self.has_error and self.context is not None and self.field_error:
+            return self.field_error
+
+    # TODO Created at / created by
```
[the remark this answers is not in the document]
Comment (kaedroho) [reply wagtail_localize/models.py:1123]: Oops, We've got that in ``last_transalted_by``. Forgot to delete the comment!

File: wagtail_localize/static_src/common/components/ImageChooser/index.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                            src={imageInfo.thumbnail.url}
+                            title={imageInfo.title}
+                            width={imageInfo.thumbnail.width}
+                            height={imageInfo.thumbnail.height}
+                        />
+                    </div>
+
+                    <ul className="actions" style={{ listStyleType: 'none' }}>
+                        {/*<li><button type="button" className="button action-clear button-small button-secondary">{gettext('Clear choice')}</button></li>
+                        <li><button type="button" className="button action-choose button-small button-secondary">{gettext('Change image')}</button></li>*/}
```
[the remark this answers is not in the document]
Comment (kaedroho) [reply wagtail_localize/static_src/common/components/ImageChooser/index.tsx:51]: Nope! thanks!

**Pull request merged**